css/index.css

```diff
@@ -111,26 +123,21 @@ p {
     align-items: center;
     justify-content: center;
 }
-
```

Comment:
Somewhere around here is where you want the colors for each individual box preferably.